gulpfile.js

```diff
@@ -188,5 +186,5 @@ function commonDepCheck(options) {
 }
 
 gulp.task('prepublish', function(done) {
-  runSequence('build-all', 'test', done);
+  runSequence('build-all', 'test:unit', done);
```

Comment:
No, we don't want to do this in prepublish at all because `npm install` runs prepublish. Added a comment.